### ISSUE TITLE
rule: inline-no-min-max-logical-dimensions

### ISSUE THREAD

**Issue by purupurupu** [purupurupu/css-noop-checker#121]
## Rule ID
`inline-no-min-max-logical-dimensions`

## What does this rule detect?
`min-inline-size`, `max-inline-size`, `min-block-size`, `max-block-size` on non-replaced inline elements have no effect. Logical equivalents of `inline-no-min-max-dimensions`.

## Example: should warn
```html
<span style="display: inline; max-inline-size: 200px;">text</span>
```

## Example: should NOT warn
```html
<span style="display: inline-block; max-inline-size: 200px;">text</span>
```

## Related CSS Properties
`display`, `min-inline-size`, `max-inline-size`, `min-block-size`, `max-block-size`

## References
- [CSS Logical Properties Level 1](https://drafts.csswg.org/css-logical/#dimension-properties): logical equivalents follow same applicability as physical counterparts
- Natural complement to existing `inline-no-logical-dimensions`
